packages/opentelemetry-core/test/utils/url.test.ts

```diff
@@ -16,29 +16,29 @@
 
 import * as assert from 'assert';
 
-import { isUrlIgnored } from '../../src';
+import { isIgnored } from '../../src';
```

Comment:
the new functionality (catcher with pattern) are missing unit tests.

Reply:
I'm going to remove catcher it is not needed for real functionality